### ISSUE TITLE
Windows 10 Home Edition Install Instructions No Longer Work

### ISSUE THREAD

**Issue by poflynn** [pi-node/instructions#27]
Alas with the latest updates to Win 10 Home this old version of VirtualBox is blocked. When you get to step 5 Windows won't run VirtualBox and instead throws up the error "This app can't run on this device" :-(

https://github.com/pi-node/instructions/wiki/(EN)-Windows-10-Home-Edition

**Comment by AlfaB16** [comment]
Yes I did face the same issue, I am still trying to get around it.

**Comment by Yorkien** [comment]
Who could help me,I usually run it unsucessful

**Comment by AlfaB16** [comment]
Hi, I've had pi test node installed on my windows 10 home edition computer
for a while now, other than having docker installed and running I was not
able to get the 10ports open. It's been a few days now that my computer is
dead after I ran into a few blue screens. So I would suggest that you quit
trying to install the node on windows home and change to windows pro
instead. All the best.
Regards.
AD.

**Comment by ihatejam** [comment]
You no longer need Docker Toolbox (which uses Virtualbox) on Windows Home edition. Docker Desktop (no need for Virtualbox) now runs on Windows Home edition. More info can be found here - 
https://docs.docker.com/docker-for-windows/install-windows-home/

**Comment by yewkhoon** [comment]
https://www.youtube.com/watch?v=HnP4yfCLt7M

**Comment by hirenshah** [comment]
> You no longer need Docker Toolbox (which uses Virtualbox) on Windows Home edition. Docker Desktop (no need for Virtualbox) now runs on Windows Home edition. More info can be found here -
> https://docs.docker.com/docker-for-windows/install-windows-home/

As I found out today, Pi Node hasn't been updated to handle this properly... it still looks for files like docker-machine.exe and therefore just fails to even begin checking the ports.

**Comment by lstfrs** [comment]
> > You no longer need Docker Toolbox (which uses Virtualbox) on Windows Home edition. Docker Desktop (no need for Virtualbox) now runs on Windows Home edition. More info can be found here -
> > https://docs.docker.com/docker-for-windows/install-windows-home/
> 
> As I found out today, Pi Node hasn't been updated to handle this properly... it still looks for files like docker-machine.exe and therefore just fails to even begin checking the ports.

Do you know how to modify name of some process or somehow make Pi node handle this? I am facing the same problem. Ports are opened but pi node doesn't check.
